source/Octopus.Cli/Commands/CompleteCommand.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Model;
+using Octopus.Cli.Util;
+
+namespace Octopus.Cli.Commands
+{
+    [Command("complete", Description = "Find the most likely subcommand completion based on a partial subcommand.")]
+    public class CompleteCommand : CommandBase, ICommand
+    {
+        private ICommandLocator commands;
+
+        public CompleteCommand(ICommandLocator commands, ICommandOutputProvider commandOutputProvider) : base(commandOutputProvider)
+        {
+            this.commands = commands;
+        }
+
+        public Task Execute(string[] commandLineArguments)
+        {
+            return Task.Run(() =>
+            {
+                Options.Parse(commandLineArguments);
+                
+                commandOutputProvider.PrintMessages = OutputFormat == OutputFormat.Default;
```

Comment:
Cant you set this to true? Dont really need `OutputFormat`, right?

Reply:
Good point! Thanks